extensions_admin/pulp_docker/extensions/admin/manifest.py

```diff
@@ -0,0 +1,118 @@
+from gettext import gettext as _
+
+from pulp.client.commands import options
+from pulp.client.commands.criteria import DisplayUnitAssociationsCommand
+from pulp.client.commands.unit import UnitCopyCommand, UnitRemoveCommand
+
+from pulp_docker.common.models import Manifest, Blob
+
+DESC_COPY = _('copies manifests from one repository into another')
+DESC_REMOVE = _('remove manifests from a repository')
+DESC_SEARCH = _('search for manifests in a repository')
+FORMAT_ERR = _('The docker manifest formatter can not process %s units.')
```

Comment:
Consider using the Python 3 compatible formatting here too. Again, not required. Also, I assume there are other places in this patch like this, so if you want to make the change I suggest find/replacing and doing it everywhere. I won't make comments on the rest of them and fill up your PR with lame comments ☺